packages/itwinui-react/src/core/DropdownMenu/DropdownMenu.tsx

```diff
@@ -26,12 +37,16 @@ export type DropdownMenuProps = {
    * Child element to wrap dropdown with.
    */
   children: React.ReactNode;
-} & Omit<PopoverProps, 'content'> &
-  Omit<CommonProps, 'title'>;
+} & Pick<
+  React.ComponentProps<typeof Popover>,
+  'visible' | 'onVisibleChange' | 'placement' | 'matchWidth'
+> &
```

Comment:
Are you not exposing all Popover props to avoid making it to complicated for the users?

Reply:
yes and to avoid issues where users start misusing random props and breaking things